pkg/registry/core/pod/storage/storage_test.go

```diff
@@ -418,6 +418,7 @@ func TestConvertToTableList(t *testing.T) {
 		{Name: "IP", Type: "string", Priority: 1, Description: v1.PodStatus{}.SwaggerDoc()["podIP"]},
 		{Name: "Node", Type: "string", Priority: 1, Description: v1.PodSpec{}.SwaggerDoc()["nodeName"]},
 		{Name: "Nominated Node", Type: "string", Priority: 1, Description: v1.PodStatus{}.SwaggerDoc()["nominatedNodeName"]},
+		{Name: "Owner", Type: "string", Priority: 1, Description: metav1.ObjectMeta{}.SwaggerDoc()["ownerReferences"]},
```

Comment:
column should probably be "Controller", not "Owner"

the injected documentation is for the entire ownerReferences array, not specifically the owner reference that is the managing controller

Reply:
done!